.github/workflows/frontend-check.yaml

```diff
@@ -0,0 +1,55 @@
+name: Cypress tests
+on: 
+  push:
+    # branches:
+    #   - main
+    # tags:
+    #   - v*
+  pull_request:
+jobs:
+  install:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Cypress install
+        uses: cypress-io/github-action@v6
+        with:
+          # Disable running of tests within install job
+          runTests: false
+          build: npm run build
+          project: ./frontend
+
+      - name: Save build folder
+        uses: actions/upload-artifact@v4
+        with:
+          name: build
+          if-no-files-found: error
+          path: ./frontend/dist
+
+  cypress-run:
+    name: Cypress run
+    needs: install
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Download the build folder
+        uses: actions/download-artifact@v4
+        with:
+          name: build        
+          path: ./frontend/dist
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        with:
+          start: npm start
+          group: 'UI-Chrome'
+          record: true
+          project: ./frontend
```

Comment:
So what's actually happening here? You're starting an optimized frontend with no backend, no `ocpperf.toml`? What's going to actually *happen* here? 😦